Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp

```diff
@@ -87,8 +87,10 @@ static GRefPtr<GstCaps> createSinkPadTemplateCaps()
     std::string emptyString;
     GRefPtr<GstCaps> caps = adoptGRef(gst_caps_new_empty());
 
-    if (!opencdm_is_type_supported(WebCore::GStreamerEMEUtilities::s_ClearKeyKeySystem, emptyString.c_str()))
+    if (!opencdm_is_type_supported(WebCore::GStreamerEMEUtilities::s_ClearKeyKeySystem, emptyString.c_str())) {
```

Comment:
Why does the OpenCDM API require you to pass an empty string here? That seems very weird.

Reply:
It doesn't. I don't know why this was done like this i.e. without specifying mime type

Reply:
OK, let's clean it up then.